### PR TITLE
feat: don’t render moderator icon if everyone is moderator

### PR DESCRIPTION
**Pull request by zbettenbuk**

### CODE REVIEW

File: react/features/base/participants/functions.js [hunk 1]
```diff
@@ -260,6 +260,25 @@ function _getAllParticipants(stateful) {
...
 }
 
+/**
+ * Returns true if all of the meeting participants are moderators.
+ *
+ * @param {Object|Function} stateful -Object or function that can be resolved
+ * to the Redux state.
+ * @returns {boolean}
+ */
+export function isEveryoneModerator(stateful: Object | Function) {
```
Comment (saghul) [inline react/features/base/participants/functions.js]: also use this on web?

**Comment by saghul** [comment]
> We have this setting in interface config.

Yeah, but mobile doeesn't read it unfortunately.

**Comment by zbettenbuk** [comment]
@damencho do you want me to put this into web too?

**Comment by damencho** [comment]
Well, I don't know. For web we have this configurable https://github.com/jitsi/jitsi-meet/blob/master/interface_config.js#L82 and if you add this it will obsolete the setting. I doubt anyone is using this settiing at the moment other than meet.jit.si and our branded deployments ...

**Comment by zbettenbuk** [comment]
@saghul Talked to @damencho, we'll not add this to web for now. although the function will be there and the knowledge that we can do this, and they will revisit this later, when need be. I'll merge the PR once the tests run

**Comment by saghul** [comment]
Sounds good.

**Pull request merged**